aksel.nav.no/website/pages/eksempler/select/description.tsx

```diff
@@ -4,10 +4,10 @@ import { withDsExample } from "@/web/examples/withDsExample";
 const Example = () => {
   return (
     <Select
-      label="Hvilket land har du bosted i?"
+      label="Velg bostedsland"
```

Comment:
Should make it easier to understand that you're supposed to choose something (not write).